src/WebJobs.Extensions.DurableTask/Correlation/DurableTaskCorrelationTelemetryInitializer.cs

```diff
@@ -0,0 +1,324 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using DurableTask.Core.Settings;
```

Comment:
Not related to this PR though, the namespace seems to be against the [naming convention](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-namespaces).

Reply:
Do you mean `<Company>.<Technology>*` rule?

Reply:
Sure. Then it might be a problem of the DurableTask framework side.